README.md

```diff
@@ -1,61 +1,274 @@
 
-##Overview
+The ViSearch PHP SDK is an open source software for easy integartion of ViSearch Search API with your application server. It provides three search methods based on the ViSearch Search API - pre-indexed search, color search and upload search. The ViSearch PHP SDK also provides an easy integration of the ViSearch Data API which includes data inserting and data removing. For source code and references, visit the github [repository](https://github.com/visenze/visearch-sdk-php).
 
-* [Setup](#setup)
-* [Initialization](#initialization)
-* [Inserting Images](#inserting-images)
-* [Removing Images](#removing-images)
-* [Searching Images](#searching-images)
+ Current stable version: 1.0.0
+ 
+ Mininum requirement: php5 and php5-curl
 
-##Setup
+##2. Setup
+You can get the soruce code of the SDK and demos from the [github repo](https://github.com/visenze/visearch-sdk-php).
 
-The ViSearch PHP SDK is dependent on php5.4+ and php5-curl.You may include the latest SDK into your project folder
+Download the [ViSearch SDK](https://github.com/visenze/visearch-sdk-php) and place the ViSearch to your project directory.
+
+Include relavant services into your project.
+
+````
+<?php
+//For ViSearch Search API
+require_once 'pathTo/ViSearch/search.php';
+//For ViSearch Data Insert API
+require_once 'pathTo/ViSearch/insert.php';
+//For ViSearch Data Remove API
+require_once 'pathTo/ViSearch/remove.php';
 ````
-require_once '../visearch-php/search.php';
-```` 
 
-##Initialization
+##3. Initialization
+To start using ViSearch API, initialize ViSearch client with your ViSearch API credentials. Your credentials can be found in [ViSearch Dashboard](https://dashboard.visenze.com):
 
 ````
+//To Use ViSearch Search API
 $service = new SearchService($access_key,$secret_key);
+//To Use ViSearch Data Insert API
+$service = new InsertService($access_key,$secret_key);
+//To Use ViSearch Data Remove API
+$service = new RemoveService($access_key,$secret_key);
+
 ````
-##Inserting Images
+##4. Indexing Images
+
+
+###4.1 Indexing Your First Images
+
+Built for scalability, ViSearch API enables fast and accurate searches on high volume of images. Before making your first image search, you need to prepare a list of images and index them into ViSearch by calling the /insert endpoint. Each image must have a unique identifier and a publicly downloadable URL. ViSearch will parallelly fetch your images from the given URLs, and index the downloaded for searching. After the image indexes are built, you can start searching for similar images using the unique identifier (link to Pre-indexed Search section), using a color (link to Color Search Section), or using another image (link to Upload Search Section).
```

Comment:
update the links, see https://github.com/visenze/visearch-sdk-java/blob/master/README.md#41-indexing-your-first-images